### ISSUE TITLE
non-partner twitch streams | fast forwarded, lag, end up with no audio

### ISSUE THREAD

**Issue by pepper-jk** [chrippa/livestreamer#1449]
Hi there,

I run into this issue for a while now and I let it slide until today, cuz it starts to annoy me.

Whenever I start watching a non-partner stream on "source" obviously, cuz there is no better option, it plays fine for a second and then fast forwardes and the sound breaks, then lag then another 5 seconds in fastforwarded and so on. I'm not quite sure it's an non-partner problem only, but I only encountered it on those streams. Riot, pokemontcg, pokemonvgc and others are fine.

I'm using archlinux [4.7.1-1-ARCH] on a i5-4670, 8GB RAM and a GTX570 (if this matters at all).

To watch I use this little script:
`livestreamer --hls-segment-threads 4 -p 'mplayer -fps 24 -zoom -x 1280 -y 720 -cache 4048 -title ""' https://www.twitch.tv/$1 "best,1080p,720p,480p,high,medium"`

The threads, fps and quality fallbacks were added today, no difference. I guess I did use double the cache a while ago, so this might not be it either. If I didn't screw up here, I guess it's due to a option on the streamers end.

Thanks in advance for your help and support,
keep up the good work guys :)

p.s. Sorry if there is already an issue for this already, did some search via google and on the issue site and didn't find anything like it.

**Comment by skulblakka** [comment]
Have you tried using a different player? The docs list a [common issue](http://docs.livestreamer.io/players.html#mplayer-tries-to-play-twitch-streams-at-the-wrong-fps) with mplayer and twitch.

**Comment by pepper-jk** [comment]
You are absolutely right mister. Stupid me. I glaced through the docs and the "commen issue" part. Next time I'll read it completely.

tl;dr:
mpv works with non-partner twitch streams as mentoined in the docs.

Thanks. :)
